source/_integrations/switchbot_cloud.markdown

```
@@ -97,6 +98,16 @@ Features:
 - turn on or off
 - get power consumption readings
 
+#### Relay Switch 2PM
+
+Features:
+displayed as two switches with the following properties or actions individually,
+- turn on or off
```

markdownlint-cli2 (0.17.2)

Lists should be surrounded by blank lines

(MD032, blanks-around-lists)